options/locale/locale_en-US.ini

```diff
@@ -1604,6 +1604,8 @@ issues.review.show_outdated = Show outdated
 issues.review.hide_outdated = Hide outdated
 issues.review.show_resolved = Show resolved
 issues.review.hide_resolved = Hide resolved
+issues.review.comment_points_to_modified_line = Points to a modified code fragment.
```

Comment:
This seems unclear. What "points"?

Reply:
Hmm I think I would leave the "pointing" part out completely. We do use "outdated" or "stale" words to refer to obsolete comments I think.

Reply:
Ok - then I'll just add  Outdated as in Conversation tab? Followed by the Show original link. Ok?

Reply:
Let's call it "Stale". We already have stale reviews, why not also stale comments.

I think you could just add a label at the end of the text:

```
<div class="ui basic label" data-tooltip-content="The code this comment refers to has changed since it was made">Stale</div>

Reply:
Perfect - i'll change it!

Reply:
Did you mean like this:

![image](https://github.com/go-gitea/gitea/assets/1135157/f66afe1b-57f8-4b71-a08c-8b828c912fec)

![image](https://github.com/go-gitea/gitea/assets/1135157/8912f944-b261-4f1a-9845-f28875c8abad)

Reply:
In Conversation tab we use the outdated label so i'd suggest we reuse this label.

![image](https://github.com/go-gitea/gitea/assets/1135157/7aa7d7f2-5544-4415-aa73-11cc7537cb26)

which would result in:

![image](https://github.com/go-gitea/gitea/assets/1135157/6d802525-2b04-4393-970a-2f7a15b314f8)

Reply:
Yeah, let's use "Outdated". Label looks good.

Reply:
Move the label after the "Show original text" please.

Reply:
![image](https://github.com/go-gitea/gitea/assets/1135157/3834c86c-44d9-4a54-acfd-d2e90e13ed68)

Reply:
For the resolved case i think a separator between resolved and Show original would be good.
Just always adding a point to root marked this conversation as resolved would do the trick. 

![image](https://github.com/go-gitea/gitea/assets/1135157/33b80fb0-a7c0-4f0b-9b1a-12dd0d59ddc4)

and if it's not outdated:
![image](https://github.com/go-gitea/gitea/assets/1135157/9322d55f-3a7d-455e-8ffa-7b9a6ac000ec)

Reply:
Done.